validator_client/src/config.rs

```diff
@@ -261,32 +267,27 @@ impl Config {
         /*
          * Web3 signer
          */
-        if let Some(s) = parse_optional::<String>(cli_args, "web3-signer-keep-alive-timeout")? {
-            config.web3_signer_keep_alive_timeout = if s == "null" {
-                None
-            } else {
-                Some(Duration::from_millis(
-                    s.parse().map_err(|_| "invalid timeout value".to_string())?,
-                ))
-            }
+        if validator_client_config.web3_signer_keep_alive_timeout == 0 {
+            config.web3_signer_keep_alive_timeout = None
+        } else {
+            config.web3_signer_keep_alive_timeout = Some(Duration::from_millis(
+                validator_client_config.web3_signer_keep_alive_timeout,
+            ));
```

Comment:
Note the breaking change here. Since I've defined `web3_signer_keep_alive_timeout as a u64, null is no longer a valid option. Users will need to use `0` instead. 

If this breaking change is too intrusive I can define `web3_signer_keep_alive_timeout` as a string and we can keep functionality as is.

Reply:
I think this is probably ok, not sure if anyone actually remove the web3signer timeout? (cc @michaelsproul)
I've added `backwards-incompat` label.

Reply:
Yeah this is probably obscure enough that we would be OK to change it. It's a bit dodgy because it flips the meaning of `0` from "timeout immediately" to "never timeout", but I doubt anyone is using 0